src/indextags.jl

```diff
@@ -1,5 +1,5 @@
 # Helper functions
-vertex_tag(v) = "$v"
+vertex_tag(v) = replace("$v", "," => "×", "(" => "", ")" => "")
 
 function vertex_tag(v::Tuple)
```

Comment:
I think this function is now unnecessary with the changes to the more generic `vertex_tag(v)`.